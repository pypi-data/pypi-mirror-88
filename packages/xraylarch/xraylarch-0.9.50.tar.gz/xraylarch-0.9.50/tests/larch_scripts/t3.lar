x = 1.0
y = 3.0

try:
    y =  7/(x-1)
except MemoryError:
    print 'mem error'
except KeyboardInterrupt:
    print  'kb error'
except IOError:
    print  'io error!!'
except ArithmeticError:
     print  'arithmetic error!!'
except:
    print 'other error'
#endtry

print y
    
