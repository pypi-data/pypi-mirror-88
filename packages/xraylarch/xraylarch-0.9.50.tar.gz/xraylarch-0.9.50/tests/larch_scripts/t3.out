arithmetic error!!
3.0
